cmds/core/init/init_plan9.go

```diff
@@ -0,0 +1,96 @@
+// Copyright 2012-2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build plan9
```

Comment:
What are the differences bere? It looks the same as the other one

Reply:
It's mostly the same and once I get this nailed down a bit better I'd like to merge them. but there are some subtleties. No $PATH in plan 9; it's the order into which you bind things onto /bin. So these paths make no sense. Little things like that.

Reply:
PTAL